packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -187,8 +187,13 @@ exports.sourceNodes = async (
         node.internal.type === `file__file`
       ) {
         try {
+          let fileUrl = node.url
+          if (typeof node.uri === `object`) {
+            // Support JSON API 2.x file URI format https://www.drupal.org/node/2982209
+            fileUrl = node.url.uri
```

Comment:
Shoudn't this be like this:
```suggestion
            fileUrl = node.uri.url
```